projects/angular-image-viewer/src/lib/angular-image-viewer.component.html

```diff
@@ -4,11 +4,11 @@
       <span>{{ imageName }}</span>
     </div>
 
-
-    <img cdkDrag [src]="src[index]" [ngStyle]="style" alt="Image not found..." [ngClass]="{'dragging': isDragOn }"
-        (cdkDragStarted)="onDragStart($event)"
-        (cdkDragEnded)="onDragEnd($event)"
-        (load)="onLoad(src[index])" (loadstart)="onLoadStart(src[index])" (error)="imageNotFound(src[index])" />
+    <div cdkDrag class="drag-container">
```

Comment:
dragDrop happens on its own div, since it applies changes to the `transform` css property